elide-integration-tests/src/test/groovy/com/yahoo/elide/tests/FilterIT.groovy

```diff
@@ -1015,6 +1017,96 @@ class FilterIT extends AbstractIntegrationTestInitializer {
     }
 
     @Test
+    public void testNonRootFilterPostfixInsensitive() {
+        int editorEdBooks = 0
+        for (JsonNode node : nullNedBooks.get("data")) {
+            if (node.get("attributes").get("editorName").asText().endsWith("d")) {
+                editorEdBooks += 1
+            }
+        }
+
+        Assert.assertTrue(editorEdBooks > 0)
+
+        /* Test Default */
+        def editorNameEndsWithd = mapper.readTree(
+                RestAssured.get("/author/${nullNedId}/books?filter[book.editorName][postfix]=D").asString())
+
+        Assert.assertEquals(0, editorNameEndsWithd.get("data").size())
+
+        editorNameEndsWithd = mapper.readTree(
+                RestAssured.get("/author/${nullNedId}/books?filter[book.editorName][postfixi]=D").asString())
+
+        Assert.assertEquals(editorEdBooks, editorNameEndsWithd.get("data").size())
+
+        /* Test RSQL Typed */
+        // TODO:
+//        editorNameEndsWithd = mapper.readTree(
+//                RestAssured.get("/author/${nullNedId}/books?filter[book]=genre==*Fiction").asString())
+//
+//        Assert.assertEquals(editorEdBooks, editorNameEndsWithd.get("data").size())
```

Comment:
It has a temporary TODO. I'm awaiting a response from this ticket to know how to support this feature with RSQL:

https://github.com/jirutka/rsql-parser/issues/25